fe/fe-core/src/test/java/com/starrocks/pseudocluster/PseudoCluster.java

```diff
@@ -0,0 +1,137 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.pseudocluster;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Maps;
+import com.starrocks.common.ClientPool;
+import com.starrocks.common.Config;
+import com.starrocks.rpc.BrpcProxy;
+import com.starrocks.rpc.LakeService;
+import com.starrocks.rpc.PBackendService;
+import com.starrocks.server.GlobalStateMgr;
+import com.starrocks.thrift.BackendService;
+import com.starrocks.thrift.HeartbeatService;
+import com.starrocks.thrift.TNetworkAddress;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class PseudoCluster {
```

Comment:
So is there any example that show how to use this PseudoCluster in ut code?

Reply:
Currently, it's not easy to use this in UT, it needs to start a new JVM to reinit some singleton class